database/factories/TravelFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * A factory for Travel.
+ *
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Travel>
+ */
+class TravelFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string,string|int|\DateTime>
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->word(),
+            'primary_contact_user_id' => User::all()->random()->id,
```

Comment:
I'd prefer something that creates a user if one doesn't exist. Or does this do that?

Reply:
This picks a random user. That's what the `DuesTransactionFactory` was already doing so I copied that.

I think this is reasonable default behavior; if something more specific is required for a test then the attribute can be overriden - https://laravel.com/docs/9.x/database-testing#overriding-attributes